dashboard/test/ui/features/teacher_tools/pl_sections.feature

```diff
@@ -22,21 +20,20 @@ Feature: Professional learning Sections
     When I select facilitator participant type
 
     # Edit Section Form
-    Then I wait to see "#uitest-section-name"
-    And I press keys "My Section of Teachers" for element "#uitest-section-name"
-    Then I wait to see "#uitest-assignment-family"
-    When I select the "ui-test-teacher-pl-course" option in dropdown "uitest-assignment-family"
-    And I press the save button to create a new section
-    And I wait for the dialog to close using jQuery
+    Then I wait to see "#sections-set-up-container"
```

Comment:
Is it worth making a shared step for these steps?

Reply:
I added this to the followup work in this task: https://codedotorg.atlassian.net/browse/TEACH-592